stat/combin/combinations_example_test.go

```diff
@@ -128,3 +108,145 @@ func ExampleIndexToCombination() {
 	// 8 [1 3 4] 8
 	// 9 [2 3 4] 9
 }
+
+func ExamplePermutations() {
+	// combin provides several ways to work with the permutationss of
+	// different objects. Permutations generates them directly.
+	fmt.Println("Generate list:")
+	n := 4
+	k := 3
+	list := combin.Permutations(n, k)
+	for i, v := range list {
+		fmt.Println(i, v)
+	}
+	// The returned values can be used to index into a data structure.
```

Comment:
I think these should be separate examples.

Reply:
Done.